### ISSUE TITLE
PATCH not supported

### ISSUE THREAD

**Issue by dmalanij** [repocho/raml-mocker#23]
I've been struggling a little bit with this until I've decided to surf into your code. Currently if you have a resource defined in RAML that supports update through PATCH, that doesn't get mocked.
In the index.js, line 123 there's a regular expression filtering it:

```
        if (method.method && /get|post|put|delete/i.test(method.method) && method.responses) {
```

With a little amendment it works fine

```
        if (method.method && /get|post|put|patch|delete/i.test(method.method) && method.responses) {
```

I'm kind of on a rush, as soon as I get a few minutes free I'll fork, fix and create Pull Request, but any case you have this reported.

**Comment by repocho** [comment]
Perfect, thanks for your contribution.
A PR would be perfect !

Thanks.

**Comment by dmalanij** [comment]
And, after an eternity, #25 is here... hope it helps!

**Comment by repocho** [comment]
Fixed in #25
